src/main/resources/vm/script/datamanagement/uploadFunctions.vm

```diff
@@ -94,71 +104,29 @@ function uploadLfnFile {
     do
         if [ "${done}" = "0" ]
         then
-            lcg-del -v -a ${OPTS} lfn:${LFN} &>/dev/null
-            lfc-ls ${LFN}
-            if [ $? = 0 ]
-            then
-                lfc-rename ${LFN} ${LFN}-garbage-${HOSTNAME}-${PWD}
-            fi
-            flag="cr"
-            lfc-mkdir -p `dirname ${LFN}`
-            lcg-cr -v ${OPTS} -d ${DEST} -l lfn:${LFN} file:${FILE} &> lcg-log
+            dirac-dms-clean-directory ${OPTS} ${LFN} &>/dev/null
+            dmkdir `dirname ${LFN}`
+            dirac-dms-add-file ${OPTS} ${LFN} ${FILE} ${DEST} &> dirac.log
         else
-            flag="rep"
-            lcg-rep -v ${OPTS} -d ${DEST} lfn:${LFN} &> lcg-log
+            dirac-dms-replicate-lfn ${OPTS} ${LFN} ${DEST} &> dirac.log
         fi
         if [ $? = 0 ]
         then
-            info "lcg-cr/rep of ${LFN} to SE ${DEST} worked fine"
+            info "Copy/Replication of ${LFN} to SE ${DEST} worked fine"
             done=`expr ${done} + 1`
-                    if [ $flag = "cr" ]
-            then
-                lcg_source=`hostname`
-                lcg_destination=$DEST
-                lcg_time=`cat lcg-log | awk '/Transfer took/ {print $3$4}'`
-                info "UploadCommand=lcg-cr Source=$lcg_source Destination=$lcg_destination Size=$size Time=$lcg_time"
```

Comment:
This is something I would like to keep for research/optimization purposes. Could you please check if/how we could parse this kind of info from Dirac logs?